api/http/transactionTypeMapper.js

```diff
@@ -0,0 +1,50 @@
+
+const transactionType = require('../../helpers/transactionTypes');
+
+function getCorrectTypeForBack(frontType) {
+
+    const typeMap = new Map();
```

Comment:
I think typeMap needs to be wrote higher, before getCorrectTypeForBack function, otherwise you will initialize typeMap on every call